msal/tests/Test.MSAL.NET.Integration/UnifiedCacheTests.cs

```diff
@@ -0,0 +1,491 @@
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//------------------------------------------------------------------------------
+
+extern alias msal;
+extern alias adal;
+
+using Test.Microsoft.Identity.LabInfrastructure;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Net;
+using System.Security;
+using System.Threading.Tasks;
+using Test.MSAL.NET.Integration.Infrastructure;
+using System.IO;
+using System.Linq;
+using System;
+
+namespace Test.MSAL.NET.Integration
+{
+    [TestClass]
+    public class UnifiedCacheTests
+    {
+        public const string ClientId = "0615b6ca-88d4-4884-8729-b178178f7c27";
+        public const string Authority = "https://login.microsoftonline.com/d34a1bb7-3481-4d5f-8b94-f3cc27bf8eac/";
+
+        public string[] MsalScopes = { "https://graph.microsoft.com/.default" };
+        public string[] MsalScopes1 = { "https://graph.windows.net/.default" };
+
+        AuthHelper authHelper = new AuthHelper();
+
+        private const string AdalResource1 = "https://graph.windows.net";
+        private const string AdalResource2 = "https://graph.microsoft.com";
+
+        private Test.Microsoft.Identity.LabInfrastructure.IUser user;
+        private SecureString securePassword;
+
+        private static System.Object cacheStorageLock = new Object();
+        private static byte[] AdalV3StateStorage;
+        private static byte[] UnifiedStateStorage;
+
+        [TestInitialize]
+        public void TestInitialize()
+        {
+            ClearCacheStorage();
+
+            user = authHelper.GetUser(
+                new Test.Microsoft.Identity.LabInfrastructure.UserQueryParameters
+                {
+                    IsMamUser = false,
+                    IsMfaUser = false,
+                    IsFederatedUser = false
+                });
+
+            new Test.Microsoft.Identity.LabInfrastructure.UserQueryParameters
+            {
+                IsMamUser = false,
+                IsMfaUser = false,
+                IsFederatedUser = false
+            };
```

Comment:
what is this doing?

Reply:
will remove

Reply:
Seems still to be around...

---
In reply to: [223460925](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1304#discussion_r223460925) [](ancestors = 223460925)